packages/calling/src/CallingClient/line/index.ts

```diff
@@ -263,6 +268,16 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
 
       return call;
     }
+    if (this.serviceData?.indicator === ServiceIndicator.GUEST_CALLING) {
```

Comment:
Nit: This needs to be an `else-if` but for some reason when i try to git push it does not reflect in the code. Will fix this soon.